### ISSUE TITLE
another utf-8 related issue

### ISSUE THREAD

**Issue by deflax** [wee-slack/wee-slack#407]
Hello :)

Im using the latest revision (at the time of writing: https://github.com/wee-slack/wee-slack/commit/0523eb9ca47b0d472f2981c700461ed78c576bff)

I'm experiencing  the following error related to unicode, which is similar to the previous stack traces but not the same:

python: stdout/stderr: Traceback (most recent call last):
python: stdout/stderr:   File "/home/afx/.weechat/python/autoload/wee_slack.py", line 585, in reconnect_callback
python: stdout/stderr:     EVENTROUTER.reconnect_if_disconnected()
python: stdout/stderr:   File "/home/afx/.weechat/python/autoload/wee_slack.py", line 282, in reconnect_if_disconnected
python: stdout/stderr:     team.connect()
python: stdout/stderr:   File "/home/afx/.weechat/python/autoload/wee_slack.py", line 1043, in connect
python: stdout/stderr:     dbg("websocket connection error: {}".format(e))
python: stdout/stderr: UnicodeDecodeError: 'ascii' codec can't decode byte 0xd0 in position 12: ordinal not in range(128)
python: error in function "reconnect_callback"
python: stdout/stderr: WS ERROR
python: stdout/stderr: WS ERROR
...

**Comment by auscompgeek** [comment]
An error whilst trying to format an error? That's meta.

This is probably the most annoying error message I've seen. Are you able to reproduce this?

**Comment by trygveaa** [comment]
This should now be fixed. Though, I haven't reproduced it. so can't confirm it. Please test, and reopen if it's still a problem.

**Comment by smaslennikov** [comment]
I'm using c2b67f978579e353ff02c59bcc879f47d8c7f0b4 and I just hit a similar error: 

1. I sent a message to a channel (with a link and two mentions)
2. Nothing happened (channel didn't update). I checked Slack webui - channel never got my message and has another there that hasn't gotten to wee_slack
3. Checked server buffer, see this:

```bash
weechat      | python: stdout/stderr: WS ERROR
weechat      | python: stdout/stderr: WS ERROR
weechat      | python: stdout/stderr: WS ERROR
weechat      | python: stdout/stderr: WS ERROR
```

I'm not even sure how I can debug this - I'm not seeing any traces.
